### PR TITLE
Using all namespaces

### PR DESCRIPTION
**Pull request by grahamcrackers**
## Description

Adding old sdk and collector packgages under the `magento-storefront` storefront namespace

### CODE REVIEW

File: packages/magento-storefront-event-collector/.npmignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# test
+coverage
+
+# misc
+.env
+
+src
+tsconfig.json
+README.md
```
Comment (aniham) [inline packages/magento-storefront-event-collector/.npmignore:17]: minor: new line

File: packages/magento-storefront-events-sdk/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+
+-   add initial order context ([d2078b2](https://github.com/adobe/magento-storefront-events-sdk/commit/d2078b260d2b2db328a93cb687376e8734722a26))
+
+-   send copy of the entire data layer context with handler rather than a reference to mdl ([37ba49a](https://github.com/adobe/magento-storefront-events-sdk/commit/37ba49af6073abe351a3c466d3080962d2b727f0)), closes [#8](https://github.com/adobe/magento-storefront-events-sdk/issues/8)
+
+### [0.0.9](https://github.com/adobe/magento-storefront-events-sdk/compare/v0.0.8...v0.0.9) (2021-03-15)
+
+### Features
+
+-   add missing unsubscribe support for generic event listeners ([f8ee42e](https://github.com/adobe/magento-storefront-events-sdk/commit/f8ee42e079e9c861be0295e47135eb3d80077a2a))
```
Comment (aniham) [inline packages/magento-storefront-events-sdk/CHANGELOG.md:349]: same with this file

File: packages/magento-storefront-event-collector/.npmignore (hunk 1 above)
Comment (grahamcrackers) [reply packages/magento-storefront-event-collector/.npmignore]: i can take care of this in a new PR

File: packages/magento-storefront-event-collector/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+* create shopper context and include it with global contexts ([e2ffb14](https://github.com/adobe/magento-storefront-event-collector/commit/e2ffb1498e3b09d9323134fe626bbdd06775edc5))
+* create shopping cart context and include it with global contexts ([0d0ffec](https://github.com/adobe/magento-storefront-event-collector/commit/0d0ffecb0a322a7ca458d4a5c3a90b9515cbf144))
+* create storefront instance context and include it with global contexts ([22b5cb1](https://github.com/adobe/magento-storefront-event-collector/commit/22b5cb1afc2637af8aee2770815e02ea99b45801))
+* include tracker global context ([7014a83](https://github.com/adobe/magento-storefront-event-collector/commit/7014a8369ed2b4e958a20379dee58f5e9924a6c9))
+* initialize adobe client data layer ([bcdc07b](https://github.com/adobe/magento-storefront-event-collector/commit/bcdc07b1a69803bb9b9f9090d4a0bca9cb2f001d))
+* initialize analytics ([6342e59](https://github.com/adobe/magento-storefront-event-collector/commit/6342e5907b02563d0b3e91458922394709c4c54b))
+* send events to snowplow ([ab6edee](https://github.com/adobe/magento-storefront-event-collector/commit/ab6edee27d9dae18a8d438ad138e30b01cbb2b63))
+
+### 0.0.1 (2021-03-17)
```
[the remark this answers is not in the document]
Comment (grahamcrackers) [reply packages/magento-storefront-event-collector/CHANGELOG.md]: i can take care of this in a new PR

File: packages/magento-storefront-events-sdk/CHANGELOG.md (hunk 2 above)
Comment (grahamcrackers) [reply packages/magento-storefront-events-sdk/CHANGELOG.md]: i can take care of this in a new PR

**Pull request merged**